.github/workflows/ci.yml

```diff
@@ -34,4 +39,4 @@ jobs:
       - run: yarn
       - name: Run iOS tests
         working-directory: ./ios
-        run: xcodebuild -project "ElectrodeReactNativeBridge.xcodeproj" -scheme "ElectrodeReactNativeBridgeTests" -destination "platform=iOS Simulator,name=iPhone 8,OS=13.4.1" test
```

Comment:
`OS=13.4.1` was this in place to make sure the bridge is compatible with the specific version and below? @liulianci

Reply:
No we don't need a specific os version

Reply:
No, it caused a build failure that was noticed and fixed in #75 - It happened when the version on the GitHub actions runner node was updated without us knowing. Removing the version will prevent these type of issues in the future.